ui/src/components/ValidatorList/ValidatorList.tsx

```diff
@@ -27,26 +30,73 @@ export default function ValidatorList({
   const [selectedRows, setSelectedRows] = useState<GridSelectionModel>([]);
   const [open, setOpen] = useState(false);
   const [loading, setLoading] = useState(false);
-
+  const [validatorSummaryURL, setValidatorSummaryURL] = useState<string>("");
+  const [summaryUrlBuildingStatus, setSummaryUrlBuildingStatus] = useState(
+    BeaconchaUrlBuildingStatus.NOT_STARTED
+  );
   const [keystoresGet, setKeystoresGet] = useState<Web3signerGetResponse>();
 
-  useEffect(() => {
-    async function getKeystores() {
-      setLoading(true);
-      const keystoresGet = await web3signerApi.getKeystores();
-      setLoading(false);
-      setKeystoresGet(keystoresGet);
+  async function getKeystores() {
+    setLoading(true);
+    const keystoresGet = await web3signerApi.getKeystores();
+    setKeystoresGet(keystoresGet);
+    setLoading(false);
+  }
+
+  async function getValidatorSummaryURL(beaconchaApi: BeaconchaApi) {
+    if (!keystoresGet) {
+      setValidatorSummaryURL("");
+      setSummaryUrlBuildingStatus(BeaconchaUrlBuildingStatus.ERROR);
```

Comment:
Never start a var with a capital letter unless it is a Global constant. Also folder should not start with capital letter

Reply:
Are you refering to the enum? As I see here (https://www.typescriptlang.org/docs/handbook/enums.html) I should change "ERROR" for "Error", but the rest is fine I think.

You also mean the ValidatorList/ directory should be validatorList/?

Reply:
https://github.com/dappnode/eth2-keymanager-frontend/commit/27d08c433e7c3c4a3dfb97a3cf2ee9df96ca8550